zipkin-common/src/test/scala/com/twitter/zipkin/storage/DependencyStoreSpec.scala

```diff
@@ -103,28 +103,25 @@ abstract class DependencyStoreSpec extends JUnitSuite with Matchers {
     val two = Endpoint(127 << 24 | 2, 9410, "trace-producer-two")
     val three = Endpoint(127 << 24 | 3, 9410, "trace-producer-three")
 
-    val trace = List(
-      Span(10L, "get", 10L, None, Some(1445136539256150L), Some(1152579L), List(
```

Comment:
I'm sure that the spark job can be written in a way that it can retain the
timestamps and that we can also find a way to allow it to run more flexibly.

For example, to test the spark job at all required refactoring. I know
because I did that work. The current form of the spark job is in no way
good enough to dictate changes here.

I'm quite sure you can open issues and find other defences that would
support dumbing down this test to allow the current spark job to pass.

I think it is a better choice to allow the spark job to become better and
pass tests on its own virtue rather than changing the tests so they pass as
is.

For example, this test we are talking about directly supports the current
pull request to support a different endTs.

The back and forth with you is more effort than fixing the code